internal/alarmnotifier/notify.go

```
@@ -0,0 +1,180 @@
+	if err := doJSON(n.client, req); err != nil {
+		return fmt.Errorf("send telegram message: %w", err)
+	}
+	return nil
+}
+
+func doJSON(client *http.Client, req *http.Request) error {
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)